host_source.go

```diff
@@ -435,10 +435,10 @@ func (h *HostInfo) Hostname() string {
 }
 
 func (h *HostInfo) ConnectAddressAndPort() string {
-        h.mu.Lock()
-        defer h.mu.Unlock()
-        addr, _ := h.connectAddressLocked()
-        return net.JoinHostPort(addr.String(), strconv.Itoa(h.port))
+	h.mu.Lock()
+	defer h.mu.Unlock()
+	addr, _ := h.connectAddressLocked()
+	return net.JoinHostPort(addr.String(), strconv.Itoa(h.port))
```

Comment:
These formatting changes are not related to this PR

Reply:
Sure, but it is definitely formatting issues, here, for example, it had spaces instead of tabs, is there a way to get these type of changes in ?

Reply:
Separate PR I guess

Reply:
Removed